src/patternfly/components/Page/page.scss

```diff
@@ -408,6 +408,10 @@
     --pf-c-page__main-section--BackgroundColor: var(--pf-c-page__main-section--m-dark-200--BackgroundColor);
   }
 
+  &.pf-m-no-padding {
```

Comment:
think it's worth noting this in the next demo?

Reply:
also add it to the docs.

Reply:
Added to the docs, what does your first comment mean?

Reply:
Oh this is a new feature of the page component, but is being bundled in this PR to create a card demo. Wondering if we should mention in the next PF demo that you can add `pf-m-no-padding` to the page sections. Up to you, it's a small thing so it doesn't matter to me. Just wanted to mention it.

Reply:
oh gotcha!